src/AnnouncementBubble/AnnouncementBubble.tsx

```diff
@@ -35,6 +35,7 @@ export const AnnouncementBubble: React.FC<AnnouncementBubbleProps> = (
           senderIcon={props.senderIcon}
           senderName={props.senderName}
           sentAtTimestamp={props.sentAtTimestamp}
+          messageTruncated={props.messageTruncated}
```

Comment:
One thing we have done at times for boolean props is try to frame the name to be slightly more obvious that it is a boolean - something like `isMessageTruncated`. That being said, this one is pretty clear! just thought i would throw it out to see which you preferred

Reply:
I can do that!